mlir/lib/Dialect/SCF/Transforms/TileUsingInterface.cpp

```diff
@@ -835,7 +835,7 @@ getUntiledProducerFromSliceSource(OpOperand *source,
 /// Implementation of fusing producer of a single slice by computing the
 /// slice of the producer in-place.
 std::optional<scf::SCFFuseProducerOfSliceResult>
-mlir::scf::tileAndFuseProducerOfSlice(
+mlir::scf::tileAndFuseProducerOfSliceImpl(
```

Comment:
Does this need to be made a publicly exposed method? It makes me squirm when a method that has "impl' in its name is exposed this way. It seems like it is leaking implementation details.

Reply:
Yes, good suggestion! I will make it `static` in next commit:)